frontend/js/src/components/Navbar.tsx

```diff
@@ -97,6 +97,9 @@ function Navbar() {
           <NavLink to="/explore/" onClick={toggleSidebar}>
             Explore
           </NavLink>
+          <NavLink to="/donors/" onClick={toggleSidebar}>
+            Donors
+          </NavLink>
```

Comment:
I think this would probably be better in the navbar-bottom element:

![image](https://github.com/user-attachments/assets/475406aa-267b-43ab-bcf9-b142510133dc)